documentation/asciidoc/topics/proc_configuring_grafana_dashboards.adoc

```diff
@@ -0,0 +1,37 @@
+[id='configuring-grafana-dashboards_{context}']
+= Configuring {brandname} Dashboards
+{ispn_operator} provides global configuration properties that let you configure Grafana dashboards for {brandname} clusters.
+
+.Procedure
+
+. Create a `ConfigMap` named `infinispan-operator-config`.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::yaml/infinispan_operator_config.yaml[]
+----
++
+. Specify a namespace that {ispn_operator} watches as the value for the `grafana.dashboard.namespace` property.
++
+[NOTE]
+====
+You can modify this property while {ispn_operator} is running.
+Deleting the value deletes the dashboard.
+Changing the value changes the {brandname} cluster to which the dashboard applies.
+====
++
+. Specify values for other `data.grafana.dashboard` fields as required.
+. Add `infinispan-operator-config` to the {ispn_operator} namespace.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ oc apply -f infinispan-operator-config.yaml
+----
++
+. Delete the {ispn_operator} pod to restart it.
```

Comment:
Ok, so this step is not necessary unless user deleted the Dashboard. In that case he can modify/recreate the configmap or delete the pod to recreate the dashboard. This step can be dropped